lib/cdo/chat_client.rb

```diff
@@ -8,6 +8,10 @@
 # This class is intended to be a thin wrapper around our chat client
 # implementation (namely Slack as of February 2017).
 class ChatClient
+  USER_GROUP_ID_MAP = {
+    'teacher-tools-on-call': 'S07FB3XSAR5'
+  }.freeze
+
```

Comment:
This should go in [Slack.rb](https://github.com/code-dot-org/code-dot-org/blob/e8c036fb7fdacff236df15680fc148f60ee5060e/lib/cdo/slack.rb#L24), not here.

Reply:
That seems more appropriate, thanks for the pointer.

I'm debating a bit between moving just this mapping to Slack.rb vs the whole logic by passing through the notify_group parameter. But, as of now the method seems to parse all parameters that are directly used by the Slack API, so keeping it as a different method.